inc/admin.php

```diff
@@ -754,6 +762,23 @@ function js_templates() {
 		include plugin_dir_path( __FILE__ ) . '../tpl/js-templates.php';
 	}
 
+
+	/**
+	 * Need to render templates for new WP 4.8 widgets when not on the 'widgets' screen or in the customizer.
+	 */
+	function js_widget_templates() {
+		$screen = get_current_screen();
```

Comment:
I'm not sure what you mean? `enqueue_admin_scripts` is also being called on pages that don't have PB "active". We need to have the templates available on pages so that when a widget is added it can be rendered properly. I think it would take a largish refactor to avoid enqueueing scripts etc on pages until we know PB is active.